plugins/ps/pre-deploy

```diff
@@ -5,4 +5,5 @@ source "$PLUGIN_AVAILABLE_PATH/ps/functions"
 
 APP="$1"
 
+get_procfile $APP
```

Comment:
Not sure what you mean @josegonzalez. We need it in both cases for generating the scale file, and especially in dockerfile deploy for discovering what commands to run.